src/main/resources/schema/ans/0.10.4/traits/trait_content_restrictions.json

```diff
@@ -29,6 +29,29 @@
         }
       },
       "required": [ "active" ]
+    },
+    "geo": {
+      "type": "object",
+      "description": "Geo-Restriction configuration that contains the restriction ids that this content should be associated with.",
+      "additionalProperties": false,
+      "properties": {
+        "restrictions": {
```

Comment:
To minimize the number of ways to "null" this field out, I'd suggest:
* Making `restrictions` a required property of geo
* Making the `minItems` the same as the `maxItems`: 1

That way there's only one way to indicate that a story has no restrictions, and that's by omitting `geo` entirely. (I.e. all saved stories today.)

We can relax these restrictions later if a use case arrives for a `geo` trait with a field other than `restrictions`